geode-core/src/test/java/org/apache/geode/internal/cache/LocalRegionUpdateTest.java

```diff
@@ -0,0 +1,151 @@
+
+    EntryEventFactory entryEventFactory = mock(EntryEventFactory.class);
+    InternalCache cache = mock(InternalCache.class);
+    InternalDistributedSystem internalDistributedSystem = mock(InternalDistributedSystem.class);
+    InternalRegionArguments internalRegionArguments = mock(InternalRegionArguments.class);
+
+    LocalRegion.RegionMapConstructor regionMapConstructor =
+        mock(LocalRegion.RegionMapConstructor.class);
+    Function<LocalRegion, RegionPerfStats> regionPerfStatsFactory = localRegion -> {
+      localRegion.getLocalSize();
+      return mock(RegionPerfStats.class);
+    };
+
+    final PoolImpl poolImpl = mock(PoolImpl.class);
+    SubscriptionAttributes subscriptionAttributes =
+        new SubscriptionAttributes(InterestPolicy.ALL);
+
+
+    when(cache.getInternalDistributedSystem()).thenReturn(internalDistributedSystem);
+    when(internalDistributedSystem.getClock()).thenReturn(mock(DSClock.class));
+
+    when(regionMapConstructor.create(any(), any(), any())).thenReturn(mock(RegionMap.class));
+
+
+    AttributesFactory<Object, Object> regionAttributesFactory = new AttributesFactory<>();
+    regionAttributesFactory.setDataPolicy(DataPolicy.NORMAL);
+    regionAttributesFactory.setPoolName("Pool1");
+    regionAttributesFactory.setConcurrencyChecksEnabled(false);
+    regionAttributesFactory.setSubscriptionAttributes(subscriptionAttributes);
+    RegionAttributes<Object, Object> regionAttributes =
+        regionAttributesFactory.createRegionAttributes();
+
+    registerInterestTracker = new RegisterInterestTracker();
+    when(poolImpl.getRITracker()).thenReturn(registerInterestTracker);
+
+    ServerRegionProxy serverRegionProxy = mock(ServerRegionProxy.class);
+    when(serverRegionProxy.getPool()).thenReturn(poolImpl);
+    LocalRegion.ServerRegionProxyConstructor proxyConstructor = region -> serverRegionProxy;
+
+    AbstractRegion.PoolFinder poolFinder = poolName -> poolImpl;
+
+    region = spy(new LocalRegion("region", regionAttributes, null, cache,
+        internalRegionArguments, internalDataView, regionMapConstructor, proxyConstructor,
+        entryEventFactory, poolFinder, regionPerfStatsFactory, disabledClock()));
+
+  }
+
+  /*
+   * As indicated by the name this code tests the basicBridgeClientUpdate method's
+   * fork where it checks to see if the interestResultPolicy is NONE, then it will
+   * call basicUpdate.
+   */
+  @Test
+  public void basicBridgeClientUpdateChecksInterestResultPolicyNoneThenUpdates() {
+    Object key = new Object();
+    Object value = new Object();
+
+    registerInterestTracker.addSingleInterest(region, key, InterestType.KEY,
+        InterestResultPolicy.NONE, true, false);
+
+    region.basicBridgeClientUpdate(null, key, value, new byte[] {'0'}, true,
+        null, true, false, new EntryEventImpl(),
+        new EventID(new byte[] {1}, 1, 1));
+
+    verify(internalDataView, times(1)).putEntry(any(), anyBoolean(),
```

Comment:
The `times(1)` here is not necessary, as that's the default behaviour of the `verify()` method with no second argument.

Reply:
I like the lack of ambiguity, but will update it.